packages/manager/modules/sms/src/sms/sms/compose/telecom-sms-sms-compose.html

```diff
@@ -142,6 +142,21 @@ <h1 data-translate="sms_sms_compose_title"></h1>
                             </div>
                         </div>
 
+                        <div class="form-group" data-ng-hide="$ctrl.isVirtualNumber()">
+                            <div data-ng-if="$ctrl.displaySenderCustomizationAdvice">
+                                <strong class="text-uppercase" data-translate="sms_sms_compose_message_advice_title"></strong>
+                                <p data-translate="sms_sms_compose_message_advice_info"></p>
+                                <p>
+                                    <span data-translate="sms_sms_compose_message_advice_description_1"></span>
+                                    <a href="#"
+                                        data-ui-sref="sms.service.senders"
```

Comment:
Can this link be injected through the routing's `resolve`

Reply:
Yes it could be. But I don't know how?

Reply:
You can find an example here with `vrackLink` : https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L67
It is injected through : https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L67
and used here : https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.html#L17